### ISSUE TITLE
Kommentare auf nächste revision übernehmen

### ISSUE THREAD

**Issue by melfja** [maplesteve/swaggertalk#28]
Wunsch für später:  
ich denke dass wir Bedarf dafür haben 'unerledigte' Kommentare auf eine nächste Revision zu übernehmen.
Es werden neue Revisionen zum Review bereitgestellt, obwohl nich alle Kommentare gelöst sind. Dann wäre diese Funktion hilfreich. Übernommene Kommentare sollten als solche gekennzeichnet sein.
Ich fürchte allerdings das dafür die 'referenzierung' der Kommentare auf eine fachliche id (Endpunktname etc)  geändert werden müsste und selbst dann könnten nur solche Kommentare übernommen werden, deren Referenz nicht umbenannt wurde. heikel.
Workarround:  die Kommentare der alten Revision müssen manuell übertragen oder erneut erfasst werden.

**Comment by maplesteve** [comment]
Ja, darüber denke ich auch schon länger nach. Mehr als die beiden von Dir vorgeschlagenen Alternativen ist mir bislang auch nicht eingefallen. Aber irgendwas in der Richtung wäre wirklich sinnvoll.
